src/ArraySetops.chpl

```diff
@@ -68,18 +68,23 @@ module ArraySetops
     // sorts and removes all values that occur
     // more than once
     proc setxor1dHelper(a: [] ?t, b: [] t) {
-      var aux = radixSortLSD_keys(concatset(a,b));
+      const aux = radixSortLSD_keys(concatset(a,b));
+      const D = aux.domain;
 
-      var sliceComp = sliceTail(aux) != sliceHead(aux);
+      const sliceComp = aux[..D.high-1] != aux[D.low+1..];
```

Comment:
@bradcray do the ranges in this line of code have performance implications? I seem to remember that they did when we wrote the initial version of the radix sort.

Reply:
My recollection / expectation is that for a slice of a distributed array `aux` that spans most of the array, this is what we'd want for locality / preserving the distributed nature of the computation.  I think in the previous cases where we expressed slices as anonymous domains (e.g., `A[{lo..hi}]`), we were focused on the case where the slice was purely or largely local to a single locale, where the current range-based slicing implementation results in too much overhead.